tasks/Resources/Provisioning/Hive/gpu-provision.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Create GPU Node In Self Managed AWS Cluster
+    Set Log Level    Info
+    ${gpu_node} =    Run Process    sh    tasks/Resources/Provisioning/Hive/AWS/provision-gpu.sh
+
+Delete GPU Node In Self Managed AWS Cluster
+    ${gpu_nodes} =    Oc Get    kind=Machine    label_selector=machine.openshift.io/instance-type=g4dn.xlarge
```

Comment:
This only considers `g4dn.xlarge` but there can be other node flavours. I think it should be a parameter, no?

Reply:
I'm planning to make this change for the next sprint also at the moment is only available on AWS